### ISSUE TITLE
Selection should not be visible during mouse interaction mode

### ISSUE THREAD

**Issue by dbkaplun** [sourcelair/xterm.js#140]
Steps to reproduce:

``` sh
npm install -g slap
slap /path/to/somefile
# use the mouse to select some text
# notice that a blue selection marker appears via xterm.js
# but other terminals use slap's built-in selection mechanism
```

**Comment by parisk** [comment]
Thanks for reporting @dbkaplun. This should be partially fixed by now after #148.

The current issue is that you get both slap's selection and the native one. I guess we should disable text selection, when `term.applicationCursor` is `true`.

**Comment by Tyriar** [comment]
Fixed in https://github.com/sourcelair/xterm.js/pull/670
